fala/templates/adviser/results.html

```diff
@@ -0,0 +1,135 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="govuk-grid-column-full">
+    <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+    <h1>Search Results</h1>
+    {% if data and data.count and data.count > 0 %}
+      {% if data.origin %}
+        {% trans page_count=data.results|length, result_count=data.count %}
+          Showing {{ page_count }} results around
+          {% pluralize %}
+          {{ result_count }} results in total. Showing {{ page_count }} results in order of closeness to
+        {% endtrans %}
+        <strong class="notranslate" translate="no">{{ data.origin.postcode }}</strong>
+        {%- if form.name.value() %}
+          {{ _('matching') }} <strong>{{ form.name.value() }}</strong>.
+        {%- else -%}
+          .
+        {% endif %}
+      {% else %}
+        {% trans result_count=data.count, page_count=data.results|length, org_name=form.name.value() %}
+          Showing {{ page_count }} result for <strong>{{ org_name }}</strong>.
+          {% pluralize %}
+          {{ result_count }} results in total. Showing {{ page_count }} results for <strong>{{ org_name }}</strong>.
+        {% endtrans %}
+      {% endif %}
+      <hr/>
+      <div class="org-list">
```

Comment:
This class isn't needed. In the CSS the rules are the following for this.

`.find-legal-adviser .search-results-list .org-list `

This selector targets elements with the class "find-legal-adviser" that have descendants elements of the following order above.

Reply:
Actually you don't need this `div` at all now, so you can delete this HTML.